src/software/gui/standalone_simulator/standalone_simulator_gui_wrapper.cpp

```diff
@@ -0,0 +1,87 @@
+#include "software/gui/standalone_simulator/standalone_simulator_gui_wrapper.h"
+
+#include <QtCore/QTimer>
+#include <QtWidgets/QApplication>
+
+#include "software/gui/standalone_simulator/widgets/standalone_simulator_gui.h"
+#include "software/proto/message_translation/ssl_geometry.h"
+
+StandaloneSimulatorGUIWrapper::StandaloneSimulatorGUIWrapper(int argc, char** argv)
```

Comment:
Could we (not in this PR) template GuiWrapper?

Reply:
Sure, what are you thinking the template arg(s) would be?

Reply:
hm, not much to template, but a lot of repeated code that could go into a super class

Reply:
hm, not really templateable but we should be able to pull out the duplicated code

Reply:
Yea will look at cleaning up / simplifying all the GUI stuff soon once simple versions of everything are running